api/api/spec/spec.yaml

```diff
@@ -10991,13 +10991,13 @@ paths:
                     - file: 0010-rules_config.xml
                       path: ruleset/rules
                       status: enabled
-                    - file: 0015-ossec_rules.xml
+                    - file: 0015-wazuh_rules.xml
                       path: ruleset/rules
                       status: enabled
-                    - file: 0016-wazuh_rules.xml
+                    - file: 0020-syslog_rules.xml
                       path: ruleset/rules
                       status: enabled
-                    - file: 0020-syslog_rules.xml
+                    - file: 0025-sendmail_rules.xml
```

Comment:
We should check the `0025-sendmail_rules.xml`, is this change related to this Epic ?

Reply:
This change is part of [#6831](https://github.com/wazuh/wazuh/pull/6831), and was validated by the framework team.